backend/services/userService.js

```diff
@@ -15,7 +15,7 @@ const getAllUsers = async () => {
 };
 
 const getOneUser = async (id) => {
-  const one = await User.findById(id);
```

Comment:
on internship my mentor told me to use findOne instead of findById , I know they work almost same, so I get used to and I replaced it just for consistency